SPECS/symlinks/symlinks.spec

```diff
@@ -41,6 +42,22 @@ install -m 644 symlinks.1 $RPM_BUILD_ROOT%{_mandir}/man1
 %{_mandir}/man1/symlinks.1*
 
 %changelog
```

Comment:
Please lint the spec to be consistent with our other specs.

Reply:
Done